### ISSUE TITLE
Animated (*.ani) cursors not being animated

### ISSUE THREAD

**Issue by GerRudi** [GerRudi/SimpleR-RotMG#14]
They just show as static cursors. 

However, this seems to be impossible as SetSystemCursor does not support this. 
A [dirty workaround](https://stackoverflow.com/a/42625013) is possible, but requires admin rights. 

There are a few AutoIt forum threads and StackOverflow questions to the winapi call in general and this really seems to be unsupported.

**Comment by GerRudi** [comment]
When monitoring the API calls used by rundll.exe when setting a cursor via the control panel:

````

Module | API
-- | --
comctl32.dll | LoadCursorW ( NULL, IDC_WAIT )
comctl32.dll | SetCursor ( 0x0000000000010009 )
main.cpl | LoadCursorW ( NULL, IDC_WAIT )
main.cpl | SetCursor ( 0x0000000000010009 )
main.cpl | LoadCursorW ( NULL, IDC_ARROW )
main.cpl | SetCursor ( 0x0000000000010005 )
main.cpl | LoadCursorW ( NULL, IDC_WAIT )
main.cpl | SetCursor ( 0x0000000000010009 )
main.cpl | LoadCursorW ( NULL, IDC_ARROW )
main.cpl | SetCursor ( 0x0000000000010005 )
comctl32.dll | SetCursor ( 0x0000000000010005 )

````

**Comment by GerRudi** [comment]
One possibility: check if custom cursor is *.ani if yes use 
````
RegWrite("HKEY_CURRENT_USER\Control Panel\Cursors", "Arrow", "REG_EXPAND_SZ", $sCursorFile)
DllCall("user32.dll", 'int', 'SystemParametersInfo', 'int', 0x0057, 'int', 0, 'int', 0, 'int', 0)
````

and on exit

````
RegWrite("HKEY_CURRENT_USER\Control Panel\Cursors", "Arrow", "REG_EXPAND_SZ", "")
DllCall("user32.dll", 'int', 'SystemParametersInfo', 'int', 0x0057, 'int', 0, 'int', 0, 'int', 0)

````

This is working, but in case of unhandled exceptions the cursor stays like this, even after reboot. 
(SetSystemCursor is only temporary)
Therefore, only use this method if the cursor ends with .ani

**Comment by GerRudi** [comment]
Patched with b8a45c73d3004586948fec28cbff26fcf034b714

**Comment by GerRudi** [comment]
Implemented with #17
